packages/teams-js/prepBetaRelease.js

```diff
@@ -86,6 +59,31 @@ function getPrefix(version) {
   }
 }
 
+/**
+ * Gets the prefix of the version that is higher amongst the inputs
+ * @param {string} currBetaVer The version currently tagged beta. (e.g. 2.0.0-beta.1)
+ * @param {string} currPkgJsonVer The version taken from the package.json. (e.g. 2.0.0)
+ * @returns The prefix of the version that is higher
+ */
+function getNextPrefix(currBetaVer, currPkgJsonVer) {
```

Comment:
Should this be called `getBetaPrefix`?

We were using "next" because the tag was next and "next-dev" because that was part of the version suffix. But now we're using beta as the tag and the version suffix, so I don't think "next" makes as much sense.

Reply:
I don't think `getBetaPrefix` is correct here. I think I should rename it to `getNewerPrefix` since this exists to determine which of the beta or package.json versions is newer. E.g if beta = 2.1.0 and package.json = 2.0.0. choose beta version. If package.json = 2.1.0 and beta = 2.0.0 then choose package.json.

Reply:
I like `getNewerPrefix` and your reasoning makes sense!